src/node.ts

```diff
@@ -83,7 +83,13 @@ export function extractNodeSpecifier(path: string): string {
  */
 function isBadCommonJs(specifier: string): boolean {
   const {name} = parseNpmSpecifier(specifier);
-  return name === "react" || name === "react-dom" || name === "react-is" || name === "scheduler";
+  return (
+    name === "react" ||
+    name === "react-dom" ||
+    name === "react-dom/client" ||
```

Comment:
The `/client` would never be part of the `name`; it’s the `path`.

Reply:
true—I shouldn't have commited that part, it is for node and … I haven't found how to make it work (at all)